pom.xml

```diff
@@ -284,37 +285,45 @@
             <dependency>
                 <groupId>com.google.errorprone</groupId>
                 <artifactId>error_prone_annotations</artifactId>
-                <version>2.15.0</version>
+                <version>2.18.0</version>
             </dependency>
 
             <!-- jetty -->
             <dependency>
                 <groupId>org.eclipse.jetty</groupId>
                 <artifactId>jetty-bom</artifactId>
-                <version>10.0.14</version>
+                <version>11.0.15</version>
                 <type>pom</type>
                 <scope>import</scope>
             </dependency>
 
             <!-- jersey -->
             <dependency>
```

Comment:
We should add these three dependencies to Airbase instead, since they are needed in consumers

Reply:
Done